### ISSUE TITLE
[Severe] Unexpected keras Convolution2D shape?

### ISSUE THREAD

**Issue by XericZephyr** [fchollet/keras#5471]
Here is how to reproduce the result:

```python
    model = Sequential()
    model.add(Convolution2D(64, 3, 3, border_mode='same', input_shape=(3, 256, 256)))
    print(model.input_shape, model.output_shape)
```

According to the [document](https://keras.io/layers/convolutional/), the output should be 
```
((None, 3, 256, 256), (None, 64, 256, 256))
```
but I got
```
((None, 3, 256, 256), (None, 3, 256, 64))
```

I have no idea why. My keras version is 1.2.2 and I tried with both tensorflow 0.10.0 and 0.12.0.

**Comment by patyork** [comment]
Your `dim_ordering` is set opposite to the data you are using.

**Comment by XericZephyr** [comment]
Got it. I found the document is targeted at theano dim ordering, however, I am using tensorflow.
